apps/scan/frontend/src/screens/scan_error_screen.tsx

```diff
@@ -26,50 +27,117 @@ export function ScanErrorScreen({
   scannedBallotCount,
   restartRequired = false,
 }: Props): JSX.Element {
-  const errorMessage = (() => {
-    if (!error) return undefined;
-    assert(
-      error !== 'double_feed_calibration_timed_out' && // Only used in double feed calibration
-        error !== 'scanner_diagnostic_failed' && // Only used in ScannerDiagnosticScreen
-        error !== 'outfeed_blocked' // Only used in ScanJamScreen
-    );
+  assert(
+    error !== 'double_feed_calibration_timed_out' && // Only used in double feed calibration
+      error !== 'scanner_diagnostic_failed' && // Only used in ScannerDiagnosticScreen
+      error !== 'outfeed_blocked' // Only used in ScanJamScreen
+  );
+
+  const {
+    title,
+    errorMessage,
+    pollWorkerMessage,
+  }: {
+    title: JSX.Element;
+    errorMessage: JSX.Element;
+    pollWorkerMessage: 'needs-help' | 'might-need-help' | 'none';
+  } = (() => {
+    if (
+      restartRequired ||
+      !error ||
+      error === 'paper_status_timed_out' ||
+      error === 'scanning_timed_out' ||
+      error === 'unexpected_paper_status' ||
+      error === 'unexpected_event' ||
+      error === 'client_error'
+    ) {
+      return {
+        title: appStrings.titleScannerError(),
+        errorMessage: (
+          <Font weight="bold">
+            {appStrings.instructionsScannerAskForRestart()}
+          </Font>
+        ),
+        pollWorkerMessage: 'none',
+      };
+    }
+
     switch (error) {
-      // Invalid ballot interpretations
+      // interpretation errors
+      case 'vertical_streaks_detected':
+        return {
+          title: appStrings.titleScannerNeedsCleaning(),
+          errorMessage: (
+            <Font weight="bold">
+              {appStrings.warningScannerNeedsCleaning()}
+            </Font>
+          ),
+          pollWorkerMessage: 'needs-help',
```

Comment:
What if we just use the actual app string here instead of a key that gets replaced later? It seems a tad more straightforward